sections/sprint-1/experiment.tex

```diff
@@ -47,76 +47,116 @@ \subsection{Setup}
+Based on the data it seems they might have a small effect on each other but not enough for it to be a problem.
+The biggest deviance in the test was only 46 cm off compared to 1.3 meters in the 1 tag test which seemed interesting, but could be because the measurements in the 3 tag experiment was made in the middle of the playing field compared to at the edge in the 1 tag experiment.
 
 \paragraph{Tag 26467}
-The overall average deviation for this tag was 11.2 cm.
-There was, however, a large deviation at \texttt{140, 90} and \texttt{(140, 120)} with 20.63 cm and 20.46 cm, respectively.
-The min and max values for $x$ and $y$ were good, and there were never any larger spikes from the actual data point than around 25 cm.
+This tag was positioned 1400 mm along the x-axis.
+The overall average deviation for this tag was 11.9 cm and it made 1.16 measurements per second.
+No measurements were further away than 26.3 cm from the actual position and the biggest average deviance for a set of measurements were only 22.26 cm.
+
 
 \paragraph{Tag 26895}
-The overall average deviation for this tag was 6.31 cm.
-The average deviation was more accurate than expected for this tag.
-For this tag, there were also no spikes larger than around 20 cm for the $x$ and $y$ values.
+This tag was positioned 1800 mm along the x-axis.
+This tag was only 9.2 cm off on average and had by far the highest average update rate at 3.4 Hz.
+Like the previous tag it also did not have any measurements with large deviances.
+The furthest measurement was 22.1 cm off and the worst average deviance for a set was 15.09 cm.
+This was by far the best performing tag in this experiment regarding both precision and update rate.
 
 \paragraph{Tag 24622}
-The overall average deviation for this tag was 17.41 cm.
-The first three points the results seemed fairly accurate, however, the grid \texttt{(220, 90)} and \texttt{(220, 120)} had a large deviation of 37.53 cm and 24.4 cm respectively.
-The min and max values for $x$ and $y$ were greater in this than the previous, with around 40 cm deviations.
+This tag was positioned 2200 mm along the x-axis.
+The overall average deviation for this tag was 14.48 cm and an update rate at 0.92 hz.
+The first three points the results seemed fairly accurate, however, the grid \texttt{(2200, 900)} had a large deviation of 22.43 cm.
+That set also had the measurement with highest deviance with 46.0 cm.
 
 \subsubsection{Precision with 5 tag}
-For this experiment, it was noted how many data points were sent in the 5-second intervals, as well as the average deviation.
 The tags were positioned at five different x-coordinates. 
-140, 160, 180, 200 and 220 were chosen as the $x$ coordinates and the $y$ coordinate was moved up by 20 cm each time starting from 0 to 120.
+1400, 1600, 1800, 2000 and 2200 were chosen as the $x$ coordinates and the $y$ coordinate was moved up by 200 mm each time starting from 0 to 120.
```

Comment:
Tror 0 to 120 skal ændres her, det er vel i cm? det skal nok lige skrives explicit eller ændres til mm